doc/user-manual/language/reflection.lagda.rst

```diff
@@ -822,6 +822,13 @@ Another form of ``unquoteDecl`` is used to declare data types:
 ``m`` is a metaprogram required to declare and define a data type ``x`` and
 its constructors ``c₁`` to ``cₙ`` using ``declareData`` and ``defineData``.
 
+.. note::
+   To debug code generated by ``unquoteDecl`` and ``unquoteDef`` it can be useful to
+   turn on the verbosity flags ``-v tc.unquote.decl:10`` (for type signatures) and
+   ``-v tc.unquote.def:10`` (for definition bodies). This will cause the generated code
+   to be printed on stdout when running from the command line or in the debug buffer
+   when loading from an editor.
```

Comment:
This should mention that these verbosities are always available even when Agda is not build with `-f debug`.
I had to look into the Agda sources to clarify this for me.